ui/src/main/java/edu/wpi/grip/ui/preview/PreviewsController.java

```diff
@@ -26,20 +29,20 @@
 @Singleton
 public class PreviewsController {
 
-    @FXML private HBox previewBox;
+    @FXML
```

Comment:
Alright. 

- Why did you change the formatting of lines 31-37? You didn't need to change anything anywhere near them.
- Why did you make `compareSockets` static and add more parameters than `Comparable.compareTo()` actually has?  Just leave it the way it is and use member variables to get the lists.
- Why did you add `final` to a bunch of variables that you otherwise didn't have to modify?
- Why did you add `getPreviews()`?

Keep in mind that I'm not necessarily arguing that all of these are bad ideas (although I believe most of them are).  I just don't think you should be doing these random unrelated refactors in this PR.